Description: fix an uninitialized variable: 
 when domain_len < 0 for instance, inside_topdomain is not initialized, but used.
Origin: vendor
Bug: http://dev.kryo.se/iodine/ticket/98
Bug-Debian: http://bugs.debian.org/596740
Forwarded: yes
Author: Samuel Thibault <sthibault@debian.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-09-20

--- a/src/iodined.c
+++ b/src/iodined.c
@@ -1567,7 +1567,7 @@
 	struct query q;
 	int read;
 	int domain_len;
-	int inside_topdomain;
+	int inside_topdomain = 0;
 
 	if ((read = read_dns(dns_fd, tun_fd, &q)) <= 0)
 		return 0;
